contracts/Vault.vy

```diff
@@ -813,6 +813,11 @@ def _calculateLockedProfit() -> uint256:
     else:
         return 0
 
+@view
+@internal
+def freeFunds() -> uint256:
```

Comment:
minor: I would use `_freeFunds` to be consistent with how the rest of internal view functions have been named